### PR TITLE
Add documentation for and by the Executive Council

### PR DESCRIPTION
**Pull request by lexnederbragt**
The executive Council has quite som documentation available, but this information is spread over several GitHub pages and Google docs. In the spirit of transparency, we would like to add this information to the Carpentries Handbook, in other words, make it public. (note that some of the pages this information links to are still private to EC members). In addition, we thereby condense our documentation to a single page (with one appendix) to make it easier for new (and current) EC members to find what they need.

This PR adds these two pages.

### CODE REVIEW

**Review by orchid00 — COMMENTED** [review]
Hi @lexnederbragt, 
I had a more detailed look at this, and fixed a few words for British English. 
Once little typo and general review. 
Thanks for setting this up.

File: topic_folders/governance/executive-council-officers.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+The Chair’s responsibilities are to:
+
+* Organize official meetings, including regularly occurring monthly meetings and additional special meetings as necessary.
+  * Set meeting times and make sure these are properly adjusted with respect to daylight savings (which does not change for all members at the same time)
+  * Send out reminder email 7-10 days before each monthly meeting (more details in the monthly minutes document)
+  * Plan dates, location, and agenda for the annual in-person meeting with input from the rest of the EC
+* Set the agenda and ensure that it is circulated in a timely fashion to allow EC members to prepare for the meeting.
+  * Review submissions to the agenda form and add these to the agenda for the meeting
+  * Invite non-members to the EC meeting as needed for specific discussions, such as staff, CI representatives
```
[the remark this answers is not in the document]
Comment (lexnederbragt) [reply topic_folders/governance/executive-council-officers.md:26]: Np. Good catch, Changed to "representatives of the fiscal sponsor"

File: topic_folders/governance/executive-council-officers.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+The Secretary maintains records for the Executive Council that are not related to either finances or the Core Team. General tasks are described in
+[the bylaws](https://docs.carpentries.org/topic_folders/governance/bylaws.html#secretary).
+
+The Secretary’s responsibilities are to:
+
+* Ensure that someone is assigned to and takes minutes during all official Executive Council meetings
+* Revise minutes of regularly occurring meetings to report decisions, policy, and passage of formal motions prior to
+circulation among the Executive Council for approval.
+* Publish meeting minutes and policy decisions
+  * publicly for the community, as described in the "Meeting conduct" document under the section ["Preparing the minutes for release"](https://docs.google.com/document/d/195omPpTOAnwNsUCqbH6S7Wpt9WDqU5joQKGMofTYDmg/edit#heading=h.yg8xqbjr7yxb)<sup>*</sup>
```
[the remark this answers is not in the document]
Comment (lexnederbragt) [reply topic_folders/governance/executive-council-officers.md:84]: I'll remove the * there. Let's keep it where it is to not confuse folks.

File: topic_folders/governance/executive-council.md [hunk 3]
```diff
@@ -1,13 +1,319 @@
...
+* Dedicating at least three volunteer hours per week toward Carpentries activities (including meeting attendance)
+
+The specific commitments expected of Executive Council members include monthly meetings (1-1.5 hours, online) and a once-yearly
+face-to-face meeting (expenses covered by The Carpentries). Minutes for these meetings are published to [a GitHub repository](https://github.com/carpentries/executive-council-info/tree/master/minutes) after
+approval by Executive Council members. We may periodically schedule additional virtual meetings to address particular topics. Officers and Executive Council
+liaisons to Committees or Task Forces may have additional commitments for meetings. Asynchronous commitments involve suggesting
+agenda items for meetings, responding to email requests for opinions, and formal voting on resolutions (as issues in this
+repository), and work resulting from being on a task force or committee representing the Executive Council.
+
+### Officers
```
[the remark this answers is not in the document]
Comment (lexnederbragt) [reply topic_folders/governance/executive-council.md:63]: The section is not only about appointment. I propose to leave the heading as it is.

**Review by lexnederbragt — COMMENTED** [review]
Thanks, @orchid00!

**Comment by maneesha** [comment]
Thanks @lexnederbragt  and @orchid00 
Is this PR ready to merge?

**Comment by lexnederbragt** [comment]
Almost. We're having an off-github conversation about it which I think will be concluded soon.

**Comment by lexnederbragt** [comment]
I feel we now are ready to merge this.

**Pull request merged**